schema/release-schema.json

```diff
@@ -2557,7 +2557,7 @@
       }
     },
     "RelatedProcess": {
-      "description": "A reference to a related contracting process: generally one preceding or following on from the current process.",
+      "description": "A reference to a related contracting (or planning) process: generally one preceding or following on from the current process.",
```

Comment:
FYI, this field can be used to refer to non-OCDS data (e.g. when the UK transitioned, they referred to TED notices). That said, I think it's fine to use "contracting (or planning) process", since the interpretation should be of any corresponding concept in TED, etc.